src/builtin.c

```diff
@@ -1355,6 +1355,7 @@ static time_t my_mktime(struct tm *tm) {
   tz = (tz = getenv("TZ")) != NULL ? strdup(tz) : NULL;
   if (tz != NULL)
     setenv("TZ", "", 1);
+  tm->is_dst = 0;
```

Comment:
Ah, so we're not fully initializing `struct tm`s...  Do we also need to set `tm_zone`?

Reply:
Would it not be better to initialize this in the callers?

Should we add `is_dst` to the jq broken-down time representation?

Reply:
False alarm!  The error I'm fixing is *already* fixed on origin/master, just not in the v1.6 release I had.

> Ah, so we're not fully initializing `struct tm`s... Do we also need to set `tm_zone`?

Actually, in `jv2tm`, a comment explicitly mentions that `tm->tm_isdst` gets zeroed by a `memset`.  I don't know why my code compiles, since my `is_dst` *should* be a typo for `tm_isdst`.

Reply:
So should we close this then?  There's nothing left to do?

Reply:
Yes.  Also, the reason my code doesn't fail is because that last-resort path of `my_mktime` isn't even compiled on my machine.  Somehow, when I did `gcc -E` to discover what my machine uses, I got the wrong answer ¯\_(ツ)_/¯.